CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unversioned
 
+- Bugfix: Downloads from the updater now follow HTTP redirects. (#4594)
```

Comment:
Replace with two dev entries:
1. Improve debug output when update checker fails
2. Add the ability to control the `followRedirect` mode for requests